src/components/Logout.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { GITHUB_LOGOUT } from "../actions/actionTypes";
+
+class Logout extends Component {
+    render() {
+        const isLoggingIn = window.location.href.indexOf('/login') > -1;
+        const { isLoggedIn } = this.props;
+
+        if (!isLoggedIn) {
+            if (isLoggingIn) {
+                return <div> </div>
+            }
+            return <Redirect to="/signup" push={true} />
```

Comment:
@FolaDee what is this supposed to do?

Reply:
It redirects to sign up if the user is not logged in